src/packs/advancements/skyvoid_one_block/data/skyvoid_advancements/function/load.mcfunction

```diff
@@ -0,0 +1,4 @@
+scoreboard players set skyvoid_advancements load.status 1

```

Comment:
The name should probably include that this is the one block advancements (e.g. `skyvoid_oneblock_advancements`). Similarly, the standard skyblock one should be named with that too (e.g. `skyvoid_skyblock_advancements`).